cdp_scrapers/instances/portland.py

```diff
@@ -257,23 +359,27 @@ def get_event_minutes(
 
         # think we will let MinutesItem.name = Matter.name
         #                   MinutesItem.description = Matter.title
+
+        minute_sections = event_page.find_all(
+            "div", class_="relation--type-agenda-item"
+        )
+        event_minute_items = []
+        for minute_section in minute_sections:
+            title_div = minute_section.find("div", class_="council-document__title")
+            matter_title = title_div.find("a").text.strip()
+            index = int(minute_section.find("h4").text.strip())
+            event_minute_item = EventMinutesItem(
+                decision=None,
+                index=index,
+                matter=self.get_matter(minute_section),
+                minutes_item=MinutesItem(name=None, description=matter_title),
+                supporting_files=self.get_supporting_files(minute_section, index),
```

Comment:
Shak can we keep this comment for `get_supporting_files()` commented out for now? I need to update that method, 1) since you are now passing in a `Tag`, not the entire web page `BeautifulSoup`; and 2) `index` is probably no longer needed; 3) I need to test the method again since you are now passing in `minute_section`, not `event_page`.

Reply:
Done.